workspaces/sandbox/plugins/sandbox/src/components/Modals/PhoneVerificationSteps/PhoneNumberStep.tsx

```diff
@@ -38,7 +38,7 @@ import InputAdornment from '@mui/material/InputAdornment';
 import CircularProgress from '@mui/material/CircularProgress';
 
 const FLAG_FETCH_URL =
-  'https://purecatamphetamine.github.io/country-flag-icons/3x2';
+  'https://catamphetamine.github.io/country-flag-icons/3x2';
```

Comment:
do we need this to be defined the sandbox-sre or can we remove it from there ?

Reply:
needed there